### PR TITLE
UIORGS-47 Fix categories dropdown

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Fix categories dropdown
## Approach
add prop `itemToString` to Categories field
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/contacts/EditContact/EditContact.js [hunk 1]
```diff
@@ -180,10 +182,11 @@ class EditContact extends Component {
                     <Field
                       component={MultiSelection}
                       dataOptions={categories.map(({ id }) => id)}
+                      formatter={this.categoriesFormatter}
+                      itemToString={this.itemToString}
```
Comment (AndreiKandratsenka) [inline src/contacts/EditContact/EditContact.js]: it seems we need to use the same approach in both places into the component or not

Comment (AndreiShumski2) [reply src/contacts/EditContact/EditContact.js]: I think it doesn’t matter here

Comment (aliaksei-chumakou) [reply src/contacts/EditContact/EditContact.js]: @AndreiKandratsenka isn't it the same approach? I see it here https://github.com/folio-org/ui-organizations/blob/9fe3ff1bf035c67edbcc4e4b108321bda1af855a/src/VendorInformation/VendorInformationForm.js#L81

Comment (AndreiKandratsenka) [reply src/contacts/EditContact/EditContact.js]: My thought was about using this.itemToString or itemToString in both places

Comment (aliaksei-chumakou) [reply src/contacts/EditContact/EditContact.js]: @AndreiShumski2 sounds good actually, there is no need to use `this, so `itemToString={itemToString}` works for both places. Could you please handle this in additional PR?

**Pull request merged**